terminus-ui/option/src/option.component.ts

```diff
@@ -53,6 +53,7 @@ export class TsOptionSelectionChange {
  * Contains properties that the options can inherit. Used by {@link TS_OPTION_PARENT_COMPONENT}
  */
 export interface TsOptionParentComponent {
+  componentName: string;
```

Comment:
@shani-terminus for the sake of knowledge sharing:

The option component needs to know whether it's parent is the autocomplete or select to know if it should show the checkbox when multiple mode is enabled. The code we originally had for this was actually giving us a false positive.. it was looking *all* the way up to the very top parent component - which because our demos are named for the item within, our parent just _happened_ to have the same name so it seemed to be working.

But when used in another app that doesn't use the same component name it doesn't work correctly. This is why we couldn't reproduce locally.

So we opted to just add a `componentName` property so the child can check directly.

Reply:
A minor correction on that explanation. The original code wasn't getting all the way to the top parent component. It was current component's parent, which in demo case happens to be`AutocompleteComponent`, while in others could be different.